folium/utilities.py

```diff
@@ -356,3 +351,38 @@ def camelize(key):
     """
     return ''.join(x.capitalize() if i > 0 else x
                    for i, x in enumerate(key.split('_')))
+
+
+def _parse_size(value):
+    try:
+        if isinstance(value, int) or isinstance(value, float):
+            value_type = 'px'
+            value = float(value)
+            assert value > 0
+        else:
+            value_type = '%'
+            value = float(value.strip('%'))
+            assert 0 <= value <= 100
+    except Exception:
+        msg = 'Cannot parse value {!r} as {!r}'.format
+        raise ValueError(msg(value, value_type))
+    return value, value_type
+
+
+def iter_points(x):
+    """Iterates over a list representing a feature, and returns a list of points,
+    whatever the shape of the array (Point, MultiPolyline, etc).
+    """
+    if isinstance(x, (list, tuple)):
```

Comment:
I don't like these convoluted nested if statements. But I guess you just copied this from branca right?

Better IMO would be to exit early. 

```
if not isinstance(x, (list, tuple)):
  raise ValueError
# happy path continues
```

Reply:
That is better. (Yep. I just copied them.)

Reply:
Also something for the next one then.